datalad/tests/utils.py

```diff
@@ -469,7 +469,12 @@ def newfunc(*arg, **kw):
         flavors_ = _get_resolved_flavors(flavors)
 
         testrepos_uris = _get_testrepos_uris(regex, flavors_)
-        assert(testrepos_uris)
+        # MIH: I don't think we can assert that there always is a testrepo
+        # for network test when NONETWORK is given, this list will be empty
+        # will change it to skipping the test. Remove on ACK.
+        #assert(testrepos_uris)
```

Comment:
There is more. None of the tests that serve a local path via http (see
decorator) work in the package build environment. I gave up on it for now.
No idea why this happens.
On Sep 13, 2015 12:55 PM, "bpoldrack" <notifications@github.com> wrote:

> In datalad/tests/utils.py
> <https://github.com/datalad/datalad/pull/200#discussion_r39346563>:
>
> > @@ -469,7 +469,12 @@ def newfunc(*arg, **kw):
> >          flavors_ = _get_resolved_flavors(flavors)
> >
> >          testrepos_uris = _get_testrepos_uris(regex, flavors_)
> > -        assert(testrepos_uris)
> > +        # MIH: I don't think we can assert that there always is a testrepo
> > +        # for network test when NONETWORK is given, this list will be empty
> > +        # will change it to skipping the test. Remove on ACK.
> > +        #assert(testrepos_uris)
>
> meh - But this means, we have to review the tests rather than adapt
> herein, I guess.
> However, whatever you need atm - just do it ;)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/datalad/datalad/pull/200/files#r39346563>.
>